client-code-gen/gen/api/famapplications-api.ts

```diff
@@ -71,11 +72,11 @@ export const FAMApplicationsApiAxiosParamCreator = function (configuration?: Con
         /**
          * gets the roles associated with an application  :param application_id: application id :param db: database session, defaults to Depends(database.get_db)
          * @summary Get Fam Application Roles
-         * @param {number} applicationId 
+         * @param {any} applicationId 
```

Comment:
Hi Ian, why we change the type for applciationId from number to any?

Reply:
Hi Catherine, the code is generated based on backend swagger spec, it is strange why it is changed. Let me try to figure it out.

Reply:
Hi Catherine, thanks for spotting this. Indeed, it is a strange case that some does get generated correctly but few don't from backend FastAPI.
I took some time to downgrade and upgrade but none can solve this issue. For now, as discussed, I manually added the missing schema type into problem endpoints and add section for note to the README.
Please have a look again. Thank you!!